test/clone.test.js

```diff
@@ -0,0 +1,18 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const clone = require('../lib/util/clone');
+
+function testCloner(expNode, new_node) {
+  it ('creates a clone of a node', function () {
+    assert.equal(clone(math.parse(expNode)), new_node);
+  });
+}
+
+describe('Clone', function () {
+  const tests = [
+    [1, 1],
+    ['x', 'x']
```

Comment:
OK fine. For the simple example we can test the value. But running the `clone()` function on a complex example, we get the child args and function `fn`. What should be actually tested for the more complex example. ?  Same goes for the test mentioned in `step.test.js`.